src/ray/raylet/worker_pool.h

```diff
@@ -90,15 +86,11 @@ class WorkerPool {
   uint32_t Size() const;
 
  protected:
-  /// Add started worker PID to the internal list of started workers (for testing).
-  ///
-  /// \param pid A process identifier for the worker being started.
-  void AddStartedWorker(pid_t pid);
-
-  /// Return a number of workers currently starting but not registered.
-  ///
-  /// \return The number of worker PIDs stored for started workers.
-  int NumWorkersStarting() const;
```

Comment:
Let's be consistent with the Mock implementation and leave this as `NumWorkerProcessesStarting` (and also replace both uses in StartWorkerProcess); removing `NumWorkerProcessesStarting` from both the mock and the WorkerPool would also be ok, but it should be consistent.

Reply:
The reason why the Mock class needs `NumWorkerProcessesStarting` is because the tests need to access it.
I'm going to replace `NumWorkerProcessesStarting()` with `starting_worker_processes_.size()` in `WorkerPoolMock.StartWorkerProcess`, and keep `WorkerPoolMock.NumWorkerProcessesStarting` for the tests.